### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by Quantum-0** [Dimpiv/create-python-app#2]
Hello, I've cloned this project to local directory on my computer. opened it, then I've tried to run utility but unsuccessfully. It doesn't work. Presumably, the reason is the absence of an executable file or script. Please, write the code ASAP!

**Comment by Quantum-0** [comment]
(just testing issues functionality)

**Comment by Dimpiv** [comment]
Yes, you are right! I get many people for writer this project and they write Big Script!

**Comment by Quantum-0** [comment]
Oh my.. Big Script? I like Big Scripts. I think I can help you to write the code for your Big Script!
How Big will be your Script? I hope it will be Very Big!

**Comment by Dimpiv** [comment]
My script will be very big! I think we can take our large scripts and move the project to a large totem script!

**Comment by Quantum-0** [comment]
Collaborative Large Totem Script. Sounds cool, I'm in!
